packages/components/psammead-social-embed/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-social-embed",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
- As these styles didn't exist in `1.0.2`, we should probably mark this as a `minor` update. Please can we amend the version to `1.1.0`?
- This version should be updated in `package-lock.json` too.